OneCryptoPkg/Library/BaseCryptCrtLibOneCrypto/OneCryptoCrt.c

```diff
@@ -0,0 +1,156 @@
+/** @file
+  C Run-Time Libraries (CRT) implementations
+  These either call the shared implementations or implement the logic itself
+  if the implementation is simple enough that the code may be throughly vetted
```

Comment:
There are no cases where the logic is implemented here and I can't imagine any that would be. We just wouldn't override those.